x-pack/plugins/ecs_data_quality_dashboard/kibana.jsonc

```diff
@@ -8,7 +8,8 @@
     "server": true,
     "browser": false,
     "requiredPlugins": [
-      "data"
+      "data",
+      "spaces",
```

Comment:
I believe "spaces" should be listed under `optionalPlugins`. For example, see https://github.com/elastic/kibana/blob/main/src/plugins/saved_search/kibana.jsonc.